client/src/i18n/locales/es/index.json

```diff
@@ -192,7 +193,7 @@
   "neAbsencesDef": "Puedes estar pagado para hasta cinco ausencias. Ausencias están incluidos en tus cálculos de ingresos.",
   "neEarnedRevenueDef": "Los ingresos que ya has ganado este mes, basado en la asistencia del niño. Esto no incluye la cuota familiar.",
   "neEstimatedRevenueDef": "Los ingresos del subsidio que predecimos que recibirás, si tienes asistencia perfecta por el resto del mes. Esto no incluye la cuota familiar.",
-  "neHoursAttendedDef": "Horas que el niño ha asistido esta semana. Si excedes tus hora máximas para la semana, no estarás pagado por el estado para esas horas adicionales.",
+  "neHoursAttendedDef": "Máximo de horas que el niño puede asistir por semana. Si excedes tus horas máximas para la semana, no estarás pagado por el estado para esas horas adicionales.",
```

Comment:
The paragraph is not well written. In Spanish, it should be: 

Si excedes tus horas máximas para la semana, el estado no te pagará por esas horas adicionales.